src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
-import IHelloProps from 'interfaces/IAppProps'
+
+import IHelloProps from 'interfaces/IAppProps';
```

Comment:
This is good. Generally we will have three groups of import.

_1. Third-party packages and react and reactdom like imports will be in first group
2. Imports of modules created by us will be in second group
3. Imports of css/scss files will be in third group_